.github/workflows/ci-workflow.yml

```diff
@@ -24,3 +25,16 @@ jobs:
       - name: "Run all tests"
         run: |
           make test
+
+  snyk:
```

Comment:
I'm curious why the Github Action workflow isn't blocking the CI pipeline where there are high severity issues showing up in the [Snyk dashboard](https://app.snyk.io/org/bcda).   Does it only check to make sure that no new vulnerabilities were added by a specific change in a feature branch (basically, ignoring existing ones)?

Reply:
I see what is going on... so okay, this checks the `go.mod` dependencies only, the errors we are running into revolves around the Dockerfiles, going to see how to add that test